source/agora/common/Config.d

```diff
@@ -189,6 +189,9 @@ public struct NodeConfig
     /// How frequent the payments to Validators will be in blocks
     public uint payout_period = 144;
 
+    /// The threshold to slash the validators did not reveal pre-images
```

Comment:
Could we document what threshold means? Something like "total number of blocks a node may refuse to reveal its preimage before it becomes slashed".

Reply:
Updated.